### PR TITLE
[Merged by Bors] - getting-started docs

### PR DESCRIPTION
**Pull request by adwk67**
# Description

Addition of getting-started documentation for a simple Airflow cluster.

## Review Checklist

- [x] Code contains useful comments
- [x] CRD change approved (or not applicable)
- [x] (Integration-)Test cases added (or not applicable)
- [x] Documentation added (or not applicable)
- [x] Changelog updated (or not applicable)
- [x] Cargo.toml only contains references to git tags (not specific commits or branches)
- [x] Helm chart can be installed and deployed operator works (or not applicable)

Once the review is done, comment `bors r+` (or `bors merge`) to merge. [Further information](https://bors.tech/documentation/getting-started/#reviewing-pull-requests)

### CODE REVIEW

File: docs/modules/getting_started/pages/first_steps.adoc [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+- the previously created secret must be referenced in `spec.credentialsSecret`
+
+
+NOTE: Please note that the version you need to specify for `spec.version` is not only the version of Apache Airflow which you want to roll out, but has to be amended with a Stackable version as shown. This Stackable version is the version of the underlying container image which is used to execute the processes. For a list of available versions please check our
+https://repo.stackable.tech/#browse/browse:docker:v2%2Fstackable%airflow%2Ftags[image registry].
+It should generally be safe to simply use the latest image version that is available.
+
+This will create the actual Airflow cluster.
+
+=== Initialization of the Airflow database
```
Comment (fhennig) [inline docs/modules/getting_started/pages/first_steps.adoc:72]:
```suggestion
== Initialization of the Airflow database
```

I think this should be one level higher, right?

File: docs/modules/getting_started/pages/first_steps.adoc [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+`connections.sqlalchemyDatabaseUri` must contain the connection string to the SQL database storing the Airflow metadata.
+
+`connections.celeryResultBackend` must contain the connection string to the SQL database storing the job metadata (in the example above we are using the same postgresql database for both).
+
+`connections.celeryBrokerUrl` must contain the connection string to the Redis instance used for queuing the jobs submitted to the airflow worker(s).
+
+The `adminUser` fields are used to create an admin user.
+Please note that the admin user will be disabled if you use a non-default authentication mechanism like LDAP.
+
+=== Airflow
```
Comment (fhennig) [inline docs/modules/getting_started/pages/first_steps.adoc:32]:
```suggestion
== Airflow
```

File: docs/modules/getting_started/examples/code/getting_started.sh.j2 [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+sleep 5
+
+echo "Checking if web interface is reachable ..."
+return_code=$(curl -s -o /dev/null -w "%{http_code}" http://localhost:8080/login/)
+if [ "$return_code" == 200 ]; then
+  echo "Webserver UI reachable!"
+else
+  echo "Could not reach Webserver UI."
+  exit 1
+fi
```
Comment (fhennig) [inline docs/modules/getting_started/examples/code/getting_started.sh.j2:103]: nice! But is it possible to also script the next parts that are going on in the UI? I mean triggering a DAG and checking if it ran through. I did that for druid: https://docs.stackable.tech/druid/nightly/getting_started/first_steps.html#_ingest_example_data

**Review by fhennig — CHANGES_REQUESTED** [review]

File: docs/modules/getting_started/pages/first_steps.adoc (hunk 1 above)
Comment (adwk67) [reply docs/modules/getting_started/pages/first_steps.adoc:72]: Yes - thanks!

File: docs/modules/getting_started/examples/code/getting_started.sh.j2 (hunk 3 above)
Comment (adwk67) [reply docs/modules/getting_started/examples/code/getting_started.sh.j2:103]: Good idea - I've added a health check and DAG run-and-check at the end.

**Review by fhennig — APPROVED** [review]
LGTM!

**Comment by adwk67** [comment]
bors merge

**Pull request closed without merging**